tests/communities/test_communities_screens_overview.py

```diff
@@ -17,7 +17,9 @@
 @pytest.mark.parametrize('params', [constants.community_params])
 def test_manage_community_screens_overview(main_screen: MainWindow, params):
     with step('Create community'):
-        main_screen.create_community(params)
```

Comment:
can you please fix the reference to test cases? https://github.com/status-im/desktop-qa-automation/blob/7547dcd830a8b44c7bc50080dfcb80fa76df391c/tests/communities/test_communities_screens_overview.py#L15
I think it is more than 1? Or if it is 1 test case, maybe rename it saying it checks all the screens?

Reply:
Added more references